File: backend/src/core/nativos/texto.js
export function mayusculas(texto) {
    return texto.toUpperCase();
}

export function minusculas(texto) {
    return texto.toLowerCase();
}

export function invertir(texto) {
    return texto.split('').reverse().join('');
}

export function concatenar(...cadenas) {
    return cadenas.join('');
}

File: src/corelibs/texto.py
"""Funciones utilitarias para manipular cadenas de texto."""


def mayusculas(texto: str) -> str:
    """Devuelve *texto* en mayúsculas."""
    return texto.upper()


def minusculas(texto: str) -> str:
    """Devuelve *texto* en minúsculas."""
    return texto.lower()


def invertir(texto: str) -> str:
    """Invierte el contenido de *texto*."""
    return texto[::-1]


def concatenar(*cadenas: str) -> str:
    """Concatena las cadenas proporcionadas."""
    return "".join(cadenas)
